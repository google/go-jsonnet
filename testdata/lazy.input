local x = {'x': error "blah"}, f = function(x) 42, z = x.x; f(x.x)