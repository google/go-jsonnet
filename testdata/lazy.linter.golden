../testdata/lazy:1:52-59 Unused variable: z

local x = {'x': error "blah"}, f = function(x) 42, z = x.x; f(x.x)


